.gitignore

```diff
@@ -5,3 +5,5 @@ __init__.py
 #*#
 *~
 *.pb
+*.root
+*replace.py
```

Comment:
Please remove these changes (also from the commit history). Additoin of `*.root` could be discussed, but it would be better done as a separate PR.